src/core/src/test/java/org/locationtech/geogig/test/integration/RepositoryTestCase.java

```diff
@@ -163,17 +163,8 @@ public final void setUp() throws Exception {
         doSetUp();
     }
 
-    /**
-     * In rare occasions a test fail for unknown reasons and it's definitely related to the
-     * temporary folder somehow resolving to the same directory in two different tests, which I
-     * thought was impossible and the whole point of {@link TemporaryFolder}. So although I didn't
-     * get to the root cause of the issue, appending a randon number to the repository directory
-     * name makes the trick for the time being.
-     */
-    private static final Random RANDOM = new Random();
```

Comment:
@groldan I think what really fixed the test issues were the changes made in GeoGigDataStoreFactoryTest in your commit here: https://github.com/locationtech/geogig/commit/b7a0b615483c8e55669b9bac8a02143a33930ac5. I'm just removing the random directory name here so that it is predictable in other tests that extend/use this class and then try to verify the repo folder.
I didn't do a commit revert because I think the envHome -> repositoryDirectory variable name refactor was good (makes it more clear what the variable is).